src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md

```diff
@@ -1,55 +1,114 @@
 ---
 title: Scheduled Jobs with Custom Clock Processes
 nav: Custom Clock Processes
-modified_at: 2021-10-19 10:00:00
+modified_at: 2023-02-01 17:00:00
 tags: task-scheduling
 index: 2
 ---
 
-With custom clock processes, you have the ability to specify custom schedule. You will also have other benefits like
-environment parity between development and production, or time precision in your task scheduling.
+Scalingo provides a feature called *Custom Clock Processes* to help you run
+tasks on a regular basis. Unlike jobs setup through the [Scalingo Scheduler]({% post_url platform/app/task-scheduling/2000-01-01-scalingo-scheduler.md %}),
+custom clock processes do not suffer from many limitations.
 
-## Definition of the Custom Clock Process
+## About Custom Clock Processes
 
-The implementation of the custom clock will vary depending on the used language. However, defining the clock process is standard and is done in the `Procfile`.
+Custom clock processes allow to define tasks so that they run periodically.
+They give you full control over the schedule, periodicity and conditions on
+which the jobs are launched, making them a powerful alternative to the Scalingo
+Scheduler.
 
-### Example
+A custom clock process is basically a background process running indefinitely
+in a container and waking up at specified interval(s) to launch some job(s).
+This background process is responsible for controlling what job(s) must be
+launched and when. It really acts as a job scheduler.
 
-Here is an example of the process definition using the Ruby library `clockwork`:
+It's usually written in the language of your choice. This means that your
+custom clock process can either share the same code base as your application's
+or a dedicated one.
 
-```yaml
-web: bundle exec puma -C config/puma.rb
-clock: bundle exec clockwork clock.rb
-```
+Custom clock processes also work with [Review Apps]({% post_url platform/app/2000-01-01-review-apps.md %}),
+just like the parent application does.
 
-## Start the Task Scheduler
+### Limitations
```

Comment:
Just like in the Scalingo scheduler doc page, I would prefer having this section later on the page.

Reply:
See https://github.com/Scalingo/documentation/pull/1915#discussion_r1097233897